assembl/scripts/pshell.py

```diff
@@ -0,0 +1,44 @@
+import sys
+
+from pyramid.util import DottedNameResolver
+from pyramid.settings import aslist
+from pyramid.scripts.pshell import PShellCommand
+
+
+def main(argv=sys.argv, quiet=False):
+    command = PShellCommandA(argv, quiet)
+    try:
+        return command.run()
+    except:
+        import pdb; pdb.post_mortem()
+
+
+class PShellCommandA(PShellCommand):
```

Comment:
Document intent please. It's to read a config file. But uncertain whether it takes an rc file, ini file, etc.